Dockerfile

```diff
@@ -1,19 +1,26 @@
-# backend/Dockerfile
+# Use an official Python runtime as a parent image
 FROM python:3.10.12
 
+# Set the working directory in the container
 WORKDIR /code
 
+# Copy the requirements file into the container at /code/
 COPY requirements.txt /code/requirements.txt
 
+# Install any dependencies
 RUN pip install --no-cache-dir -r /code/requirements.txt
 
+# Copy the rest of the application code to /code/
 COPY ./app /code/app
 
-# Local development key set
-# ENV TYPES: dev, production
-# When set to dev, API Key on endpoint requests are just 'dev'
-# When set to production, API Key on endpoint requests are the actual API Key
-
+# Set environment variables
 ENV PYTHONPATH=/code/app
 
-CMD ["fastapi", "dev", "app/main.py", "--host=0.0.0.0", "--port=8000"]
+# Copy the service account key file
+COPY local-auth.json /code/local-auth.json
```

Comment:
Omit since not needed, would be resolved when others finish the CI/CD pipeline for deployment